File: Fondamenti_programmazione_II/Python/Esami/Gestione_progetti_ricerca/Progetto.py
class Progetto:
    def __init__(self, codice, titolo, coordinatore, organizzazione, importo, tipo):
        self._codice = codice
        self._titolo = titolo
        self._coordinatore = coordinatore
        self._organizzazione = organizzazione
        self._importo = importo
        self._tipo = tipo 

    def getTipo(self):
        return self._tipo

    def getCodice(self):
        return self._codice

    def __str__(self):
        return self._tipo +"\t"+ self._titolo +"\t"+ str(self._codice) +"\t"+ self._coordinatore +"\t"+ self._organizzazione +"\t"

File: Fondamenti_programmazione_II/Java/EserciziEsame/GestioneProgettiRicercaProf/Progetto.java
package EserciziEsame.GestioneProgettiRicercaProf;

public class Progetto {
    protected int codice;
    protected String titolo, coordinatore, organizzazione;
    protected double importo;

    public Progetto(int codice, String titolo, String coordinatore, String organizzazione, double importo){
        this.codice=codice;
        this.titolo=titolo;
        this.coordinatore=coordinatore;
        this.organizzazione=organizzazione;
        this.importo=importo;
    }

    public String getTitolo(){
        return titolo;
    }

    public int getCodice(){
        return codice;
    }

    public String toString(){
        return titolo+"\t"+codice+"\t"+coordinatore+"\t"+organizzazione+"\t";
    }
}
